src/contracts/Protocol/BountyQuestion.sol

```diff
@@ -48,7 +57,11 @@ contract BountyQuestion is Ownable, OnlyApi {
         return _questionIdCounter.current();
     }
 
-    struct QuestionData {
+    function getQuestionData(uint256 questionId) public view returns (QuestionData memory) {
+        return questionData[questionId];
+    }
+
+    struct QuestionMetaData {
```

Comment:
to keep consistent probably change url to uri as I had to change this on front end as well